src/app/public/modules/card/card.component.ts

```diff
@@ -48,6 +59,16 @@ export class SkyCardComponent implements AfterContentInit, OnDestroy {
     this.subscription = this.titleComponent.changes.subscribe(() => {
       this.showTitle = this.titleComponent.length > 0;
     });
+
+    this.inlineDeleteComponent.forEach(item => {
+      item.setType(SkyInlineDeleteType.Card);
```

Comment:
Do we expect inline delete components to be present at the point `ngAfterContentInit` fires? This never actually gets hit in your visual test, because the inline-delete is controlled by an `ngIf` that is controlled by the user click. Is this even necessary?

Reply:
I actually already had a unit test for this. I don't really think we need  a visual test with this since we have a unit test and there is nothing visually different. Thoughts?

Reply:
No, I just meant unit test. If you've got it covered, then great 👍